File: Famoser.MassPass.Data/Entities/Communications/Request/UpdateRequest.cs
﻿using System;
using Famoser.MassPass.Data.Entities.Communications.Request.Base;

namespace Famoser.MassPass.Data.Entities.Communications.Request
{
    public class UpdateRequest : ApiRequest
    {
        public Guid ContentId { get; set; }
        public Guid CollectionId { get; set; }
        public string VersionId { get; set; }
    }
}

File: Famoser.MassPass.Api/src/Models/Request/UpdateRequest.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 28/05/2016
 * Time: 17:41
 */

namespace Famoser\MassPass\Models\Request;


use Famoser\MassPass\Models\Request\Base\ApiRequest;

class UpdateRequest extends ApiRequest
{
    public $ContentId;
    public $VersionId;
}